packages/components/organisms/f-header/CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v4.12.0
+------------------------------
+*March 16, 2021*
+
+### Fixed
+- Fixed typo in Italian header
+
 
 Latest (add to next release)
```

Comment:
is it okay to add my changelog comment above this one? Or should I edit the previous one ("Latest (add to next release)") somehow?

Reply:
You should combine the changelog from here with your versioned changes.